src/actions/component.js

```diff
@@ -32,6 +34,35 @@ export function getClipboardRepresentation (entity, componentName) {
   return AFRAME.utils.styleParser.stringify(diff);
 }
 
+/**
+ * Get the component docs link
+ * @param  {string} componentName Component's name
+ * @return {string}               URL to the documentation
+ */
+export function getComponentDocsUrl (componentName) {
+  if (AFRAME.components[componentName]) {
+    // Returns link from the core components
+    return 'https://aframe.io/docs/' + AFRAME.version + '/components/' +
+      (componentName === 'camera' ? '' : componentName.toLowerCase() + '.html');
+  }
+  return null;
```

Comment:
It could be `null, false, undefined...` something to easy check against that it has a valid url.